plugins/woocommerce/tests/e2e-pw/tests/merchant/create-shipping-zones.spec.js

```diff
@@ -90,6 +80,12 @@ test.describe( 'WooCommerce Shipping Settings - Add new shipping zone', () => {
 				'local_pickup'
 			);
 			await page.click( '#btn-ok' );
+			await page.waitForLoadState( 'networkidle' );
+			await expect(
+				page
+					.locator( '.wc-shipping-zone-method-title' )
+					.filter( { hasText: 'Local pickup' } )
+			).toBeVisible();
```

Comment:
We need to add this wait and this assertion to make sure that the shipping zone does not get saved as "Everywhere" in the nightly smoke test site.  `localhost` is fast enough to save the shipping zone properly with the correct name, but not the nightly smoke test site.